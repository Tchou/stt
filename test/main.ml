include Atom_test
include Char_test
include Int_test

let () = Test_utils.summary "global"
