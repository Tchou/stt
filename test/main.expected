Test: 'Char' (3/3)
      Serie: 'Char.union': (5/5)
             
             'Char.union.1': PASSED
                 expected: 'A'--'Z'
                   result: 'A'--'Z'
             'Char.union.2': PASSED
                 expected: 'A'--'Z'
                   result: 'A'--'Z'
             'Char.union.3': PASSED
                 expected: 'A'--'E'
                   result: 'A'--'E'
             'Char.union.4': PASSED
                 expected: Char
                   result: Char
             'Char.union.5': PASSED
                 expected: 'A'--'Ĭ'|'Ş'--'Ɛ'|'ƒ'--'΄'
                   result: 'A'--'Ĭ'|'Ş'--'Ɛ'|'ƒ'--'΄'
             
      
      Serie: 'Char.inter': (5/5)
             
             'Char.inter.1': PASSED
                 expected: Empty
                   result: Empty
             'Char.inter.2': PASSED
                 expected: 'D'--'G'
                   result: 'D'--'G'
             'Char.inter.3': PASSED
                 expected: 'A'|'E'
                   result: 'A'|'E'
             'Char.inter.4': PASSED
                 expected: Char
                   result: Char
             'Char.inter.5': PASSED
                 expected: 'F'--'d'|'ÿ'|'ǂ'--'̠'
                   result: 'F'--'d'|'ÿ'|'ǂ'--'̠'
             
      
      Serie: 'Char.diff': (6/6)
             
             'Char.diff.1': PASSED
                 expected: Empty
                   result: Empty
             'Char.diff.2': PASSED
                 expected: Char
                   result: Char
             'Char.diff.3': PASSED
                 expected: Empty
                   result: Empty
             'Char.diff.4': PASSED
                 expected: '\u0'--'@'|'['--'\u10ffff'
                   result: '\u0'--'@'|'['--'\u10ffff'
             'Char.diff.5': PASSED
                 expected: 'A'--'E'|'Ā'--'Ĭ'|'̡'--'΄'
                   result: 'A'--'E'|'Ā'--'Ĭ'|'̡'--'΄'
             'Char.diff.6': PASSED
                 expected: 'e'--'þ'|'Ş'--'Ɛ'|'ƒ'--'ǁ'
                   result: 'e'--'þ'|'Ş'--'Ɛ'|'ƒ'--'ǁ'
             
      
      
Test: 'Int' (3/3)
      Serie: 'Int.union': (5/5)
             
             'Int.union.1': PASSED
                 expected: Int
                   result: Int
             'Int.union.2': PASSED
                 expected: Int
                   result: Int
             'Int.union.3': PASSED
                 expected: Int
                   result: Int
             'Int.union.4': PASSED
                 expected: 65--300|350--400|402--900
                   result: 65--300|350--400|402--900
             'Int.union.5': PASSED
                 expected: 0--39
                   result: 0--39
             
      
      Serie: 'Int.inter': (6/6)
             
             'Int.inter.1': PASSED
                 expected: Int
                   result: Int
             'Int.inter.2': PASSED
                 expected: 10--100
                   result: 10--100
             'Int.inter.3': PASSED
                 expected: Empty
                   result: Empty
             'Int.inter.4': PASSED
                 expected: -1--1
                   result: -1--1
             'Int.inter.5': PASSED
                 expected: 70--100|255|450--800
                   result: 70--100|255|450--800
             'Int.inter.6': PASSED
                 expected: Empty
                   result: Empty
             
      
      Serie: 'Int.diff': (7/7)
             
             'Int.diff.1': PASSED
                 expected: Empty
                   result: Empty
             'Int.diff.2': PASSED
                 expected: Int
                   result: Int
             'Int.diff.3': PASSED
                 expected: Empty
                   result: Empty
             'Int.diff.4': PASSED
                 expected: -*---1|1--*
                   result: -*---1|1--*
             'Int.diff.5': PASSED
                 expected: 65--69|256--300|801--900
                   result: 65--69|256--300|801--900
             'Int.diff.6': PASSED
                 expected: 101--254|350--400|402--449
                   result: 101--254|350--400|402--449
             'Int.diff.7': PASSED
                 expected: 1|3|5|7|9|11|13|15|17|19|21|23|25|27|29|31|33|35|37|39
                   result: 1|3|5|7|9|11|13|15|17|19|21|23|25|27|29|31|33|35|37|39
             
      
      
SUMMARY:
        Series: 6/6
        Tests: 34/34
        
